content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -278,7 +281,7 @@ The affinity term is applied to the union of the namespaces selected by `namespa
 Note that an empty `namespaceSelector` ({}) matches all namespaces, while a null or empty `namespaces` list and 
 null `namespaceSelector` means "this pod's namespace".
 
-This feature is beta and enabled by default. You can disable it via the
+This feature is in the beta version and enabled by default. You can surely disable it via the
```

Comment:
Surely isn't required here. I agree with @kbhawkey reg. keeping the original text

```suggestion
This feature is in the beta version and enabled by default. You can disable it via the
```